### DISCUSSION TITLE
4.7.6

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#886] [Announcements]
#### ADDED
- 支持屏蔽彩色弹幕 #870
- 完善快捷键自定义功能，新增快捷键行为：切换字幕开关 #863
#### CHANGED
- 打开下载页时不重新加载下载项 #852
- 打开图片时不再遮挡状态栏 #871 #552 #272
    - 感谢 @ProJend
- 全局mica半透明背景样式调整 #871 #96
    - 感谢 @ProJend
- 去除多余的白色焦点边框 #871
    - 感谢 @ProJend
- 状态栏顶部拖拽窗口 #858
    - 感谢 @ProJend
#### FIXED
- 修复sql脚本错误 #851
- 修复FFmpegInterop播放器播放视频偶现播放中突然跳转视频结尾或无声音 #841
- 修复分区->番剧->连载动画页的港澳台番剧打开失败 #853
- 安装教程补充 #857
- 修复部分视频媒体流加载错误 #859
- 修复UP主个人主页报错 #869
- 开启使用下载索引设置项后下载视频无字幕 #860
- 帮助页新增 Windows 11 24H2 杜比全景声无声说明 #872
- 新标签页已设置背景图不存在时提示

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.6'>4.7.6</a>.</em>

**Comment by KennethWuWu** [comment]
大佬x86版本的详情页和评论区会变成透明的，是因为我的设备屏幕太小了（8寸），还是因为bug啊？

**Reply by ywmoyue** [reply]
> 大佬x86版本的详情页和评论区会变成透明的，是因为我的设备屏幕太小了（8寸），还是因为bug啊？

bug，试试最新临时版本4.7.6.702

**Reply by KennethWuWu** [reply]
最新版的也是这样。

**Reply by ywmoyue** [reply]
@KennethWuWu 麻烦重新发一下截图，我看你之前发的截图没有发出来

**Reply by KennethWuWu** [reply]
![mmexport1731049341214.jpg](https://github.com/user-attachments/assets/b2184cb4-742e-4e17-9203-a72929b65c86)

![mmexport1731049339043.jpg](https://github.com/user-attachments/assets/ff0e8875-0078-4d82-a5bf-58deda5e6663)

就像这样，详情页完全是透明的，基本看不到字。

**Reply by ywmoyue** [reply]
@KennethWuWu 麻烦新开一个bug的issue，说明问题、操作系统、设备架构等情况 https://github.com/ywmoyue/biliuwp-lite/issues/new/choose 我看看能不能复现，不能复现的话我得想别的办法处理

**Reply by ywmoyue** [reply]
如果想保持正常使用的话，可以先回退4.7.4
